requirements-llm-dev.in

```diff
@@ -0,0 +1,2 @@
+-r ./requirements-llm.txt
```

Comment:
I wonder if we actually need this for running e2e tests? IIUC garak should not be needed locally to launch the pods etc. I'm only mentioning it in case garak takes a long time to install, if not, it's probably better to KISS and leave it as-is.

Reply:
Oh, great point! Let me check that

Reply:
Done